### ISSUE TITLE
HaxeUI seems to be broken

### ISSUE THREAD

**Issue by 0xFireball** [ianharrigan/haxeui#356]
Unfortunately, it looks like HaxeUI is broken right now, at least on my system! With the latest haxelib package, I get a compile error, and with the latest development version (haxelib git) I get a runtime error about `invalid field access: push`. Can anyone help me out?

Thanks!

**Comment by ianharrigan** [comment]
Hi, what version are you using? haxelib? Git?

**Comment by ianharrigan** [comment]
Actually, just reread the actual comment. :)

What version of openfl / lime? Haxe?

**Comment by 0xFireball** [comment]
My haxelibs are all up-to-date as of today:
Haxe/Haxelib version: 3.3.0
OpenFL version: 3.6.1
Lime: 2.9.1
HaxeUI (official release, not git): 1.8.18

**Comment by ianharrigan** [comment]
Right, ok, so i think the issue is that you are using haxe 3.3. The git version of haxeui is compatible but i havent wrapped that up in to a haxelib release. Ill do that now.

**Comment by ianharrigan** [comment]
Ok, i've just created `1.8.19` haxelib release which should include the haxe 3.3 fixes. Let me know if that fixes the issue.

Cheers,
Ian

**Comment by 0xFireball** [comment]
Thanks! It compiles now, but crashes immediately (i've only tested it on Neko so far, but here's the trace)

```
Invalid field access : push
Called from openfl/_legacy/display/DisplayObjectContainer.hx line 31
Called from haxe/ui/toolkit/core/RootManager.hx line 58
Called from haxe/ui/toolkit/core/Toolkit.hx line 349
Called from Main.hx line 11
Called from C:\HaxeToolkit\haxe\std/neko/_std/Reflect.hx line 58
Called from ApplicationMain.hx line 112
Called from openfl/_legacy/Lib.hx line 122
```

This causes the program to crash immediately. This is with an extremely simple test application, here's the source:

```
import haxe.ui.toolkit.core.Macros;
import haxe.ui.toolkit.core.Toolkit;
import haxe.ui.toolkit.core.Root;
import haxe.ui.toolkit.controls.Button;
import haxe.ui.toolkit.themes.*;

class Main {
    public static function main() {
        Toolkit.theme = new DefaultTheme();
        Toolkit.init();
        Toolkit.openFullscreen(function(root:Root) {
            var button:Button = new Button();
            button.text = "HaxeUI!";
            root.addChild(button);
        });
    }
}
```

**Comment by ianharrigan** [comment]
Anything different from cpp or flash?

**Comment by 0xFireball** [comment]
I'll try it with cpp on windows.

I don't have any love for flash, especially with all the security issues. I think flash should die, so I don't have anything to play flash on my computer, I even removed it from all my browsers, so sorry, I can't test on flash.

**Comment by 0xFireball** [comment]
I just tested it on cpp for windows, it seems to work all right. Looks like it's just Neko that's broken now.

**Comment by Cor-cor** [comment]
I'm having this issue too.

haxe 3.3.0-rc.1
haxeui 1.8.20
openfl 3.6.1
lime 2.9.1

cpp for windows and flash are working, but neko build crashes with the same trace.

**Comment by ianharrigan** [comment]
Ok, cheers. Ill look into it shortly... I dont have haxe 3.3 installed at the moment however, but i can see if its an issue on 3.2.

**Comment by ianharrigan** [comment]
I cant reproduce this in haxe 3.2.1... Does anyone have 3.2.1 where this issue exists?

**Comment by puggsoy** [comment]
Just wanna say that I also have this issue, using Haxe 3.3.0 and the recommended versions of OpenFL and Lime. I'd prefer to not have to downgrade Haxe to use this.
